navbar-service.js

```diff
@@ -28,6 +28,15 @@ function factory(config) {
   // exposes menus for configuration
   service.menus = {};
 
+  // service.display may be a string or an array of strings
+  // valid values: all, brand, menu, toolbar
+  service.display = 'all';
+  service.siteTitle = null;
+
+  $rootScope.$on("$routeChangeSuccess", function() {
```

Comment:
Are we guaranteed that `$routeChangeSuccess` fires before a route component's controller runs? We don't want their change to get overwritten.